apps/web/next.config.mjs

```diff
@@ -196,6 +196,7 @@ const config = {
         __SENTRY_TRACING__: false,
       }),
     )
+    webpackConfig.module.rules.push({ test: /\.svg$/, use: ["@svgr/webpack"] });
```

Comment:
I think we don't need importing svg file, unless you need to set some props into it.
next/image or just image loading is better than bundle it into js

Reply:
did this just as im used to using svg imports directly as the svg component over Next Image which is why i added it for rendering Fiat logos (which i have to collect more of). but yeah sure can revert this and do that instead.